src/components/ListboxContainer.tsx

```diff
@@ -11,40 +11,51 @@ interface ListboxContainerProps {
 }
 
 const ListboxContainer = ({ layout, app, listboxOptions }: ListboxContainerProps) => {
-
-  const fieldName = getFieldName(layout);
+  const [fieldName, setFieldName] = useState<string | undefined>(undefined);
   const [listboxInstance, setListboxInstance] = useState<stardust.FieldInstance>();
-  const elRef = useRef();
+  const elRef = useRef<HTMLElement>();
+
+  useEffect(() => {
+    setFieldName(getFieldName(layout));
+  }, [layout?.title || layout?.qListObject?.qDimensionInfo?.qFallbackTitle]);
 
   useEffect(() => {
+    if (!fieldName) return;
     const nebbie = embed(app, {
-    //   context: {
-    //     language: translator.language,
-    //   },
+      //   context: {
+      //     language: translator.language,
+      //   },
     });
-    nebbie.field(fieldName).then((inst: any) => setListboxInstance(inst));
-  }, []);
+    nebbie.field(fieldName).then((inst: stardust.FieldInstance) => setListboxInstance(inst));
+  }, [fieldName]);
 
   useEffect(() => {
     if (!elRef.current || !listboxInstance) {
       return undefined;
```

Comment:
Probably my code, but should return a function here. The lint should have detected this inconsistent return.